1. Mexiko Create filenames filename-mappings and infofiles.py

```diff
@@ -326,13 +326,15 @@ def create_infofiles(row, filenames_file, not_ok_file):
             personnamn_not_even = True
             infotext += "|depicted people     = " + row["Personnamn / avbildad"] + "\n"
         else:
+            if "Linné, Sigvald" in row["Personnamn / avbildad"]:
```

Comment:
The old logic was correct but likely failing due to the accent on the "e".

I would revert and instead try just changing `if j + " " + i in ("Sigvald Linne", "Sigvald Linné")`.

Two changes though:
* There should be a `/` after `[[:d:Q5959424|Sigvald Linné]]` not a `\n`
* don't set `linne_category = True` since this is what you use to indicate that he was the photographer